typings/vscode.proposed.chatParticipantAdditions.d.ts

```diff
@@ -0,0 +1,361 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
```

Comment:
I think these APIs are stable now.  Let me see if we can move to the official typing file and ditch the .d.ts files.